src/copick/impl/filesystem.py

```diff
@@ -709,7 +712,11 @@ def _query_overlay_segmentations(self) -> List[CopickSegmentationFSSpec]:
     def ensure(self) -> None:
         """Ensure the run directory exists, creating it if necessary."""
         if not self.fs_overlay.exists(self.overlay_path):
+            print(self.overlay_path)
```

Comment:
Arg, i hate to say it, but at some point the prints should probably become `logging` calls

Reply:
ooops, forgot a debug print statement. I'll remove that. This method will change as part of the random access anyways.